### DISCUSSION TITLE
ok, how does the screen chip even work??

### DISCUSSION THREAD

**Discussion by XenBogo** [UkrainianBanderasCat/Digital-Logic-Sim2#85] [Q&A]
I've tried many times to get an output from the screen chip but i never got anything.
Does anybody know how to?

**Comment by AlphaCodingPilot** [comment]
Use the first 4 input pins for the x-coordinate and the next 3 pins for the y-coordinate of the pixel you want to change (starting from  bottom left). The    next 3 inputs are for the rgb-values for the color of the pixel. The next inputs represents whether you want to save the change. Finally the last input is for cleaning the screen. For example 000011101010 whould make the top left pixel green.
